code/modules/mob/living/carbon/human/human.dm

```diff
@@ -797,7 +797,10 @@
 		override = dna.species.override_float
 	..()
 
-/mob/living/carbon/human/vomit(lost_nutrition = 10, blood = 0, stun = 1, distance = 0, message = 1, toxic = 0)
+/mob/living/carbon/human/vomit(lost_nutrition = 10, blood = 0, stun = 1, distance = 0, message = 1, toxic = 0, player_triggered = 0)

```

Comment:
The general code standard is "forced" for a var name showing whether the player intended it or not.